### ISSUE TITLE
Add more clarification to the LTS doc

### ISSUE THREAD

**Issue by jsumners** [pinojs/pino#815]
I like the language about legacy versions and how they are are supported in https://github.com/googleapis/google-cloud-node#supported-nodejs-versions . I think we could adopt some form of it.

**Comment by mcollina** [comment]
I'm ok with that language as long as we clarify that we drop old Node.js versions in a major.

**Comment by jsumners** [comment]
Indeed. I’m suggesting keeping what we currently have and extending it with a version of what is linked.

**Comment by mcollina** [comment]
PR? Definitely good

**Comment by jsumners** [comment]
Yeah. This issue is a TODO for me in the future or easy fodder for a new contributor.
